pyproject.toml

```diff
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["setuptools", "unasync", "tokenize-rt"]
+requires = ["setuptools", "unasync @ https://github.com/spyoungtech/unasync/archive/refs/heads/unasync-remove.zip", "tokenize-rt"]
```

Comment:
Once https://github.com/python-trio/unasync/pull/75 is merged and the new unasync version is published to pypi, we can remove this change